### PR TITLE
update branding for 3.1 sevicing

### PR DESCRIPTION
**Pull request by Anipik**

### CODE REVIEW

**Review by ericstj — APPROVED** [review]

File: eng/Versions.props [hunk 1]
```diff
@@ -2,13 +2,13 @@
 <Project>
```
Comment (safern) [inline eng/Versions.props:2]: Do we have to tell it to not produce stable versions somewhere?

**Review by safern — APPROVED** [review]

File: eng/Versions.props (hunk 1 above)
Comment (Anipik) [reply eng/Versions.props:2]: No, we never produce a stable package from coreclr

**Review by wtgodbe — APPROVED** [review]

**Comment by Anipik** [comment]
@jashook @hoyosjs  can you please take a look at the failures here ? they are not linked to the changes in this pr and this is a high priority

**Comment by jashook** [comment]
I would not block on the CI. Something is wrong, and needs to be investigated, but this change did not cause it.

**Comment by wtgodbe** [comment]
@jashook is someone tracking investigating whatever was wrong here? We'll still need to fix the issue ASAP for 3.1.1

**Pull request merged**